server/src/main/java/io/deephaven/server/barrage/BarrageStreamGenerator.java

```diff
@@ -775,10 +923,10 @@ private ByteBuffer getSubscriptionMetadata(final SubView view) throws IOExceptio
         final int nodesOffset = metadata.endVector();
 
         BarrageUpdateMetadata.startBarrageUpdateMetadata(metadata);
-        BarrageUpdateMetadata.addNumAddBatches(metadata,
-                LongSizedDataStructure.intSize("BarrageStreamGenerator", view.numAddBatches));
-        BarrageUpdateMetadata.addNumModBatches(metadata,
-                LongSizedDataStructure.intSize("BarrageStreamGenerator", view.numModBatches));
+        BarrageUpdateMetadata.addNumAddBatches(metadata, LongSizedDataStructure.intSize("BarrageStreamGenerator",
+                (view.numAddRows + view.batchSize() - 1) / view.batchSize()));
+        BarrageUpdateMetadata.addNumModBatches(metadata, LongSizedDataStructure.intSize("BarrageStreamGenerator",
+                (view.numModRows + view.batchSize() - 1) / view.batchSize()));
```

Comment:
This isn't accurate, is it? It's probably better to not set any value for this, what do you think?

Reply:
Until we implement the `barrage` repo update, we need to provide this.  It is accurate unless the message size limit is exceeded or we cross a chunk boundary both of which are unlikely in regular use.